.circleci/config.yml

```diff
@@ -34,6 +34,10 @@ jobs:
           name: go vet
           command: make vet
 
+      - run:
+          name: go test
+          command: make test
+
```

Comment:
This is from your testing and shouldn't be added, right?

Reply:
Nope, I wanted to add this so we catch it next time. If another person rights a test that does something before the sdk test helper `resource.Test(...)` block, we won't catch it normally. We also may forget about about this whole thing months from now.

Reply:
I'll actually add a comment here on why this is added.